pkg/proxy/apis/config/validation/validation.go

```diff
@@ -283,6 +284,24 @@ func validateKubeProxyNodePortAddress(nodePortAddresses []string, fldPath *field
 	return allErrs
 }
 
+func validateIPVSTimeout(config kubeproxyconfig.KubeProxyIPVSConfiguration, fldPath *field.Path) field.ErrorList {
```

Comment:
do we need to move it to an independent function?
Seems that the conntrack timeouts are validated in the same function
> config.TCPCloseWaitTimeout.Duration
> config.TCPEstablishedTimeout.Duration

Reply:
I think it makes to have a function for this given the current way we validate the ipvs configuration